### PR TITLE
Product detail page

### PR DESCRIPTION
**Pull request by tanveertkd**
Added a product detail page for individual items. To view, you can go to products>view product or just click on this [link](https://deploy-preview-6--ti-gear-shop.netlify.app/routes/product-item.html)

### CODE REVIEW

File: css/product-item.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+.product-item-body {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    padding: 8rem;
+}
+
+.section-left {
+    width: 500px;
+    height: 500px;
```
Comment (kshitijthatte) [inline css/product-item.css:10]: You can use rem units here

File: css/product-item.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+.product-image {
+    width: 100%;
+    max-width: 350px;
+}
```
Comment (kshitijthatte) [inline css/product-item.css:20]: Here also, you can use rem units.

**Review by kshitijthatte — APPROVED** [review]
Looks great!

**Review by Amandeep-sinha — APPROVED** [review]

File: css/product-item.css (hunk 2 above)
Comment (tanveertkd) [reply css/product-item.css:20]: Will keep these suggestions in mind going forward. Thank you!

File: routes/product-item.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+                        <i class="fa-brands fa-cc-visa item-payment-icn"></i>Zero Interest EMI on select cards
+                    </p>
+                </div>
+                
+                <p class="item-stock">
+                    In Stock!
+                </p>
+
+                <div class="product-button-container">
+                    <button class="prod-btn btn-cta-buy">Buy Now</button>
```
Comment (KaustubhM1997) [inline routes/product-item.html:127]: You could add hover to these buttons as well to keep the hover styling consistent as you've added it for the goto button. The rest looks good!

**Pull request merged**